welcome.html

```diff
@@ -14,6 +16,10 @@
           <P class="version-tag">v1.0.0</P>
         </div>
       </div>
+      <input class="click-icon" type="checkbox" id="click-icon">
```

Comment:
yes, it is. because of "x" icon 

<img width="659" alt="Screenshot 2024-07-10 at 1 22 37 PM" src="https://github.com/scheme-dojo/scheme-dojo.github.io/assets/143992531/0d55e5ce-e99b-4572-a469-e07ce8086896">

Reply:
The photos are easy to understand. thank you.
For now, let's change the name to click-icon. 
Most icons can be clicked right? 
What about the `cancel button` or the name `x-icon`?

However, I think it would be better to use JavaScript to manage the state. 
If you have a reason why you really want to use CSS, that's fine. . .